src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/ApprovedConsentOrderAboutToSubmitHandler.java

```diff
@@ -77,9 +77,9 @@ private void generateAndPrepareDocuments(String authToken, CaseDetails caseDetai
                                              CaseDetails caseDetailsBefore, CaseDocument latestConsentOrder) {
         String caseId = caseDetails.getId().toString();
         log.info("Generating and preparing documents for latest consent order, Case ID: {}", caseId);
-
         Map<String, Object> caseData = caseDetails.getData();
         StampType stampType = documentHelper.getStampType(caseData);
+        LocalDate approvalDate = documentHelper.getConsentDateOfOrder(caseData);
```

Comment:
This variable is only used within a subsequent if statement. You should therefore define it within that scope.